src/vs/workbench/contrib/mergeEditor/browser/view/editors/inputCodeEditorView.ts

```diff
@@ -152,29 +152,32 @@ export class InputCodeEditorView extends CodeEditorView {
 								return [
 									baseRange.input1Diffs.length > 0
 										? action(
-											'mergeEditor.takeInput1',
-											localize('mergeEditor.takeInput1', 'Take Input 1'),
+											'mergeEditor.acceptInput1',
+											localize('mergeEditor.accept', 'Accept ') + model.input1Title,
```

Comment:
We cannot do string-concat with NLS strings, use `localize('mergeEditor.accept', 'Accept {0}', model.input1Title)` instead. That allows to place the title elsewhere in other languages, like "Foo akzeptieren"

Reply:
Thanks, didn't know NLS supports placeholders!